<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lab_02 computer graphics</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>591</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>591</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="group_Move">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Move</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QLineEdit" name="lineEdit_MoveX">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_MoveY">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_Move">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>83</y>
          <width>155</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Move</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_MoveX">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>38</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>dx:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_MoveY">
        <property name="geometry">
         <rect>
          <x>108</x>
          <y>38</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>dy:</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_Move_2">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Scale</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QLineEdit" name="lineEdit_ScaleX">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ScaleY">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_Scale">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>83</y>
          <width>155</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ScaleX">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>38</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>kx:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ScaleY">
        <property name="geometry">
         <rect>
          <x>108</x>
          <y>38</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ky:</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="group_Move_3">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Turn</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QLineEdit" name="lineEdit_TurnX">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_TurnY">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>60</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_Turn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>83</y>
          <width>155</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Turn</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_TurnX">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>38</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ox:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_TurnY">
        <property name="geometry">
         <rect>
          <x>108</x>
          <y>38</y>
          <width>45</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>oy:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
